index.js

```diff
@@ -33,16 +34,53 @@ if (process.env.EVENT_TYPES) {
     options.eventTypes = ["deployment_info", "deployment_success", "deployment_failed", "deployment_step_success", "deployment_step_failure", "group_change_success", "group_change_failed", "failed_health_check_event", "health_status_changed_event", "unhealthy_task_kill_event"]
 }
 
+if (process.env.APP_ID_REGEXS) {
+    // Use environment variable
+    if (process.env.APP_ID_REGEXS.indexOf(",") > -1) {
```

Comment:
Is splitting by `,` for regexes? Will only work it the regexes themselves don't contain a `,`